docs/cs-manual/source/tables.rst

```diff
@@ -0,0 +1,177 @@
+Tables
+######
+
+If you have launched OpenMW CS already and played around with it for a bit, you
+have probably gotten the impression that it contains lots of tables. You’d be
+spot on: OpenMW CS is built around using tables. This does not mean it works
+just like a spreadsheet application, though. It would be more accurate to think
+of databases instead. Due to the vast amounts of information involved with
+Morrowind tables just made the most sense. You have to be able to spot
+information quickly and be able to change them on the fly. Let’s browse through
+the various screens and see what all these tables show.
+
+
+Used Terms
+**********
+
+Record
+   An entry in OpenMW CS representing an item, location, sound, NPC or anything
+   else.
+
+Reference, Referenceable
+   When an item is placed in the world, it does not create a new record each
+   time. For example, the game world might contain a lot of exquisite belts on
+   different NPCs and in many crates, but they all refer to one specific
+   record: the Exquisite Belt record. In this case, all those belts in crates
+   and on NPCs are references. The central Exquisite Belt record is called a
+   *referenceable*. This allows modders to make changes to all items of the
+   same type. For example, if you want all exquisite belts to have 4000
+   enchantment points rather than 400, you will only need to change the
+   referenceable Exquisite Belt rather than all exquisite belts references
+   individually.
+
+Some columns are recurring throughout OpenMW CS, they show up in (nearly) every
+table.
+
+ID
+   Each item, location, sound, etc. gets the same unique identifier in both
+   OpenMW CS and Morrowind. This is usually a very self-explanatory name. For
+   example, the ID for the (unique) black pants of Caius Cosades is
+   ``Caius_pants``. This allows you to manipulate the game in many ways. For
+   example, you could add these pants to your inventory by simply opening the
+   console and writint: ``player- >addItem Caius_pants``. Either way, in both
+   Morrowind and OpenMW CS, the ID is the primary way to identify all these
+   different parts of the game.
+
+Modified
+   This column shows what has happened (if something has happened) to this
+   record. There are four possible states in which it can exist.
+
+Base
+   This record is part of the base game and is in its original state. Usually
```

Comment:
This is the wrong definition of base. Base is the collection of all the content files loaded except the one you are currently editing.